packages/gatsby-remark-autolink-headers/README.md

```diff
@@ -75,6 +76,7 @@ module.exports = {
               className: `custom-class`,
               maintainCase: true,
               removeAccents: true,
+              isIconAfterH: true,
```

Comment:
i suggest to add more 5 chars, then i is self explaining what means `H`
```suggestion
              isIconAfterHeader: true,
```

Reply:
Great idea!